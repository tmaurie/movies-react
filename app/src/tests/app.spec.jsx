import React from 'react';
import { render } from '@testing-library/react';

import App from '../app';

describe('App', () => {
  it('should math the expected snapshot', () => {
    const app = render(<App />);
    expect(app.asFragment()).toMatchSnapshot();
  });
});
