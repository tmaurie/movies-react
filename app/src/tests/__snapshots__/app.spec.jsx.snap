// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should math the expected snapshot 1`] = `
<DocumentFragment>
  <div>
    Hello World !
  </div>
  <div>
    Let's do a nice looking app :)
  </div>
</DocumentFragment>
`;
